frontend/packages/kubevirt-plugin/src/components/vms/vm.tsx

```diff
@@ -172,10 +175,15 @@ export const VirtualMachinesPage: React.FC<VirtualMachinesPageProps> = (props) =
   ];
 
   const flatten = ({ vms }) => getLoadedData(vms, []);
+  const ko: any = kindObj(kind);
```

Comment:
leaving `ko` with no type results in `Property 'namespaced' does not exist on type '{}'.`

leaving `ko: K8sKind` results in `Type '{}' is missing the following properties from type 'K8sKind': abbr, kind, label, labelPlural, and 2 more.`

any suggestion? @suomiy @mareklibra

Reply:
I think you can use `VirtualMachineModel` directly here

Reply:
\+ we could add this kind of logic to vm-template as well

Reply:
the `VirtualMachineTemplatesPage` uses `ListPage` and doesn't have this bug

Reply:
I see, good